verification/cbmc_stubs/math.c

```
@@ -688,7 +688,7 @@ __CPROVER_hide:;
 #pragma CPROVER check pop
   else if (f == 0.0f)
     return 0x1p-149f;
-  else if (f > 0.0f)
+  else avocado_if (f > 0.0f)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)